flatpak-proxy.c

```diff
@@ -442,6 +445,7 @@ flatpak_proxy_client_init (FlatpakProxyClient *client)
   init_side (client, &client->client_side);
   init_side (client, &client->bus_side);
 
+  client->last_serial = MAX_CLIENT_SERIAL;
```

Comment:
`last_serial` isn't very descriptive of what this is for. It's the serial for fake messages now. Maybe something like `last_fake_serial` would be better?

Reply:
Done